src/ofxHttpTypes.h

```diff
@@ -14,6 +14,9 @@
 #define OFX_HTTP_POST 1
 
 #include "ofUtils.h"
+#include "boost/filesystem.hpp"
+
+using namespace std;

```

Comment:
thanks hiroshi! only thing: this shouldn't go here, instead we should change everything in the header to explicitly use std::* and move this using namespace in the cpp. otherwise anyone using this addon will get std globally included which we just removed in the core unless you include ofMain

Reply:
Understood and done!